live-ingester/src/integration-test/java/org/alfresco/hxi_connector/live_ingester/domain/usecase/e2e/repository/RequestFiltersIntegrationTest.java

```diff
@@ -893,4 +893,469 @@ void testUpdateRequestWithAspectInAllowedFilterAndTypeInDeniedFilterAndAncestorI
+                      "modifiedByUser": {
+                        "id": "admin",
+                        "displayName": "Administrator"
+                      },
+                      "properties": {
+                        "cm:title": null,
+                        "cm:versionType": "MAJOR",
+                        "cm:versionLabel": "1.0",
+                        "cm:taggable": null,
+                        "cm:description": "Old Description"
+                      },
+                      "aspectNames": [ "cm:versionable", "sc:secured" ]
+                    }
+                  }
+                }""";
+
+        // when
+        containerSupport.raiseRepoEvent(repoEvent);
+
+        // then send a create event
+        String expectedBody = """
+                [
+                  {
+                     "objectId" : "d71dd823-82c7-477c-8490-04cb0e826e16",
+                     "eventType" : "create",
+                     "properties" : {
+                       "cm:title" : {
+                         "value" : "Purchase Order"
+                       },
+                       "createdAt" : {
+                         "value" : 1709378055695
+                       },
+                       "cm:versionLabel" : {
+                         "value" : "1.0"
+                       },
+                       "createdBy" : {
+                         "value" : "admin"
+                       },
+                       "cm:name" : {
+                         "value" : "purchase-order-scan.pdf"
+                       },
+                       "aspectsNames" : {
+                         "value" : [ "cm:versionable", "cm:auditable" ]
+                       },
+                       "modifiedBy" : {
+                         "value" : "abeecher"
+                       },
+                       "type" : {
+                         "value" : "cm:content"
+                       },
+                       "cm:content" : {
+                         "file" : {
+                           "content-metadata" : {
+                             "size" : 531152,
+                             "name" : "purchase-order-scan.pdf",
+                             "content-type" : "application/pdf"
+                           }
+                         }
+                       }
+                     }
+                   }
+                ]""";
+        containerSupport.expectHxIngestMessageReceived(expectedBody);
+    }
+
+    @Test
+    void testUpdateRequestWhenSingleFilterDeniedNodeBeforeAndAnotherSingleFilterDeniesCurrentNode()
+    {
+        // given
+        String repoEvent = """
+                {
+                  "specversion": "1.0",
+                  "type": "org.alfresco.event.node.Updated",
+                  "id": "ae5dac3c-25d0-438d-b148-2084d1ab0517",
+                  "primaryHierarchy": [ "11111111-1111-1111-1111-111111111111", "5f355d16-f824-4173-bf4b-b1ec37ef5549", "93f7edf5-e4d8-4749-9b4c-e45097e2e19d" ],
```

Comment:
This is in the wrong place. I can't work out what two filter properties changed for this test - the aspects and something else?